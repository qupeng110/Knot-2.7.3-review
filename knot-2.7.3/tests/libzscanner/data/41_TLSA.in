$ORIGIN	.
$TTL	1

; OK
@	TLSA	0	0	0	00		; The simplest variant
@	TLSA	255	255	255	00		; Maximal numbers
@	TLSA	0	0	0	0102 00 FF	; Hex string with blank spaces inside
@	TYPE52	\# 4 00000000				; TYPE + Hexadecimal rdata
@	TYPE52	0       0       0       00		; TYPE
@	tlsa	0       0       0       00		; Type in lower-case

; KO
@	TLSA
@	TLSA						; Empty rdata
@	TLSA	\# 0					; Hex empty rdata
@	TLSA	256	0	0	00		; Algorithm overflow
@	TLSA	0	256	0	00		; Flags overflow
@	TLSA	0	0	256	00		; Iterations overflow
@	TLSA	0	0	0	0		; Hex block length must be multiple of 2
@	TLSA	0	0	0	0x		; Bad hex char
@	TLSA	0	0	0			; Missing item
