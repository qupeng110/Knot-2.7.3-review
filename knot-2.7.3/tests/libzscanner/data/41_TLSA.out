OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0034
RDATA=00000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0034
RDATA=FFFFFF00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0034
RDATA=000000010200FF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0034
RDATA=00000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0034
RDATA=00000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0034
RDATA=00000000
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_NUMBER8_OVERFLOW
------
WARNG=ZS_NUMBER8_OVERFLOW
------
WARNG=ZS_NUMBER8_OVERFLOW
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_BAD_HEX_CHAR
------
